measure

```diff
@@ -247,48 +380,102 @@ class NewRelicDriver(Measure):
         self.start_timer()
 
     def measure(self):
-        skip_sleep = os.environ.get('SKIP_SLEEP', False)
-        self.progress_message = 'Measurements started'
-        self.print_progress()
+        sleep_fn = time.sleep # overriden when slo provided
+        skip_sleep = os.environ.get('SKIP_SLEEP', False) # NOTE: this skips SLO checking as well as sleeps
 
         control = self.input_data.get('control', {})
         warmup = int(control.get('warmup', DFLT_WARMUP))
         duration = int(control.get('duration', DFLT_DURATION))
         delay = int(control.get('delay', DFLT_DELAY))
 
+        slo_check = control.get('userdata', {}).get('slo_check')
+        slo = control.get('userdata', {}).get('slo')
+
+        if slo_check == 1 and slo is None:
```

Comment:
the 'slo_check' var is used in this exact construct (slo_check == 1) and only in this construct. Maybe, set it initially as "slo_check = get(userdata, ...) == 1" and then use it as a proper boolean.

NOTE that this construct means 'slo_check=1' enables SLO check, any other value disables it (e.g, slo_check=2, or slo_check="yes" would both disable it). Is this the actual intent?

(same question for fast_fail_config.disabled != 1 elsewhere in the code)

Reply:
For slo_check, I added a [simple validator](https://github.com/opsani/servo-newrelic/blob/48e0b9c49000134c7231952b1af667297a99acdc/measure#L395-L396) since the list of possible values is quite small. 

`fast_fail_config.disabled` is validated by a [pydantic conint](https://github.com/opsani/servo-newrelic/blob/48e0b9c49000134c7231952b1af667297a99acdc/measure#L177) (I don't do the same for slo_check because it seemed like overkill to define a pydantic class to parse a single flag)